cirls.Rproj

```diff
@@ -19,4 +19,6 @@ LineEndingConversion: Posix
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageBuildArgs: --compact-vignettes
+PackageCheckArgs: --as-cran
```

Comment:
Added automatically because of my RStudio settings, but I think useful (especially the second one)